Tasks/DeployVisualStudioTestAgent/task.json

```diff
@@ -6,18 +6,19 @@
     "helpMarkDown": "[More Information](https://github.com/Microsoft/vso-agent-tasks/blob/master/Tasks/DeployVisualStudioTestAgent/README.md)",
     "category": "Test",
     "visibility": [
-                "Build"
+                "Build",
+                "Release"
                   ],
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 7
+        "Patch": 8
     },
     "demands": [
 
     ],
-    "minimumAgentVersion": "1.85.0",
+    "minimumAgentVersion": "1.86.0",
```

Comment:
some how I feel RM is in preview and if user faces an issue we can ask them to update version. But we shouldn't update the agent  used in CI flow. it is ok to ask people to use new agent for RM scenarios.